docs/general/community_packages.md

```diff
@@ -0,0 +1,39 @@
+![SeAT](http://i.imgur.com/aPPOxSK.png)
+
+# Community Packages
+
+Below is a list of packages contributed by the community. These packages normally follow the same installation procedure, however, its best you consult the documentation of the package itself in case there is anything special you need to get it working.
+
+## Generic package installation
+Packages will normally come in the form of a composer package that you need to include in your SeAT install, as well as a Service Provider that you need to bootstrap. So, generically speaking, installing a package will mean that you:
+
+- *Require* the package via composer:
+
+```bash
+composer require <package vendor>/<package-name>
+```
+
+- Bootstrap the packages service provider [here](https://github.com/eveseat/seat/blob/master/config/app.php#L184).
```

Comment:
Packages for SeAT 3.0 should have Laravel Autodiscover f.e.
```
"extra": {
        "laravel": {
            "providers": [
                "Herpaderpaldent\\Seat\\SeatGroups\\GroupsServiceProvider"
            ]
        }
    },
```
I would suggest the following:
```
# enter `composer require <package vendor>/<package-name>
# Publish `php artisan vendor:publish --force --all`
# run migration (if needed) `php artisan migrate`
```